src/main/java/com/sap/sgs/phosphor/fosstars/data/json/UnpatchedVulnerabilitiesStorage.java

```diff
@@ -170,42 +168,14 @@ private static UnpatchedVulnerabilitiesStorage check(UnpatchedVulnerabilitiesSto
    */
   public static void main(String... args) throws IOException {
     UnpatchedVulnerabilitiesStorage storage = UnpatchedVulnerabilitiesStorage.load();
+
     storage.add(
         "https://github.com/odata4j/odata4j",
-        new Vulnerability(
-            "https://nvd.nist.gov/vuln/detail/CVE-2014-0171",
-            "XML external entity (XXE) vulnerability",
-            CVSS.v2(5.0),
-            Collections.emptyList(),
-            Resolution.UNPATCHED,
-            UNKNOWN_INTRODUCED_DATE,
-            UNKNOWN_FIXED_DATE));
-    storage.add(
```

Comment:
Are we not adding any more test CVEs?

Reply:
This code is not actually for adding data for tests. It's just a convenient method for updating the JSON file with unpatched issues.